src/ape_ethereum/ecosystem.py

```diff
@@ -152,12 +152,19 @@ def as_dict(self):
 class Receipt(ReceiptAPI):
     def raise_for_status(self, txn: TransactionAPI):
         """
-        Raises :class`~ape.exceptions.OutOfGasError` when the
-        transaction failed and consumed all the gas.
+        Raise an error for the given transaction if the transaction has failed.
```

Comment:
```suggestion
        Raise an error for the given transaction, if the transaction has failed.
```

Reply:
Hmm I am not sure a comma is needed here.